app/iht/views/ihtHelpers/return_link.scala.html

```diff
@@ -15,6 +15,15 @@
 *@
 
 @(label: String = "",
-link: Option[Call] = None)
+link: Option[Call] = None,
+hash: String = "")
 
-<a id="return-button" href="@link">@Html(label)</a>
+@url = @{
+    if(hash.length > 0){
+        link.getOrElse("") + "#" + hash
```

Comment:
The # character is used in a few places - it would be a bit more efficient to store he hash character in the Constants file and just refer to it.